addons/mass_mailing_sale/models/utm.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class UtmCampaign(models.Model):
+    _inherit = 'utm.campaign'
+
+    ab_testing_winner_selection = fields.Selection(selection_add=[
+        ('sale_quotation_count', 'Quotations'),
```

Comment:
Note that if current user has no access to SO, sale_quotation_count is set to 0 . Moreover, I fear performance issue as this field is computed one mailing at a time, and is not really implemented to be efficient.